src/couch_scanner/src/couch_scanner_plugin.erl

```diff
@@ -601,7 +609,7 @@ default_shards(Mod, _F, _A) when is_atom(Mod) ->
     end.
 
 default_db_opened(Mod, _F, _A) when is_atom(Mod) ->
-    fun(St, _Db) -> {ok, St} end.
+    fun(St, _Db) -> {0, [], St} end.
```

Comment:
It’s really a nice feature to let users to customize traversal!

Should we add a configuration option to `default.ini` instead of hardcoding `0`?
If we can modify it at runtime, is it possible to trigger the scanner manually via `remsh`?